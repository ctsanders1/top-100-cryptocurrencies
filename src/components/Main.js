import React from 'react'
import CurrencyFilter from '../containers/CurrencyFilter'

const Main = () => (
  <div>
    <CurrencyFilter />
  </div>
)

export default Main
