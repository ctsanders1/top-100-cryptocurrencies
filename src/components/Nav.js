import React from 'react'
import { NavLink } from 'react-router-dom'

const Nav = () => (
  <ul className="nav justify-content-center my-5">
    <li className="nav-item mr-1">
      <NavLink
        exact
        to="/"
        className="btn btn-outline-primary"
        activeClassName="disabled"
      >
        Home
      </NavLink>
    </li>{' '}
    <li className="nav-item ml-1">
      <NavLink
        exact
        to="/settings"
        className="btn btn-outline-primary"
        activeClassName="disabled"
      >
        Settings
      </NavLink>
    </li>
  </ul>
)

export default Nav
