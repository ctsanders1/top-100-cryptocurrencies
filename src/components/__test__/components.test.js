import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Main from '../Main'
import Nav from '../Nav'

it('Main renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Main />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Nav renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Nav />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
