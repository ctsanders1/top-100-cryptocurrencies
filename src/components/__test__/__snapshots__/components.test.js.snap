// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<div
  className="container"
>
  <ul
    className="nav justify-content-center my-5"
  >
    <li
      className="nav-item mr-1"
    >
      <a
        aria-current="page"
        className="btn btn-outline-primary disabled"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
     
    <li
      className="nav-item ml-1"
    >
      <a
        aria-current={null}
        className="btn btn-outline-primary"
        href="/settings"
        onClick={[Function]}
        style={undefined}
      >
        <img
          alt="gears icon"
          src="two-big-gears.svg"
          title="settings"
          width="20"
        />
      </a>
    </li>
  </ul>
</div>
`;

exports[`Nav renders correctly 1`] = `
<ul
  className="nav justify-content-center my-5"
>
  <li
    className="nav-item mr-1"
  >
    <a
      aria-current="page"
      className="btn btn-outline-primary disabled"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
  </li>
   
  <li
    className="nav-item ml-1"
  >
    <a
      aria-current={null}
      className="btn btn-outline-primary"
      href="/settings"
      onClick={[Function]}
      style={undefined}
    >
      <img
        alt="gears icon"
        src="two-big-gears.svg"
        title="settings"
        width="20"
      />
    </a>
  </li>
</ul>
`;
